src/components/Breakpoints/Breakpoints.tsx

```diff
@@ -0,0 +1,11 @@
+import { BreakpointsProps } from './Breakpoints.types';
+import { useBreakpoints } from './useBreakpoints';
+
+export function Breakpoints({ points, children }: BreakpointsProps) {
+  const [current] = useBreakpoints();
+  const active = [].concat(points).some((point) => current.includes(point));
+  if (!active) {
+    return null;
+  }
+  return <>{children}</>;
```

Comment:
❗️
Условия, блоки переменных, ретурны отбиваем пустой строкой)
```suggestion
  const active = [].concat(points).some((point) => current.includes(point));
  
  if (!active) {
    return null;
  }
  
  return <>{children}</>;
```